src/Validation/Validation.php

```diff
@@ -522,15 +522,53 @@ public static function localizedTime($check, $type = 'datetime', $format = null)
 
     /**
      * Boolean validation, determines if value passed is a boolean integer or true/false.
+     * The list of what is considered to be boolish values, may be set via $booleanValues.
      *
      * @param string $check a valid boolean
+     * @param string $booleanValues list of valid boolish values, defaults to `[true, false, 0, 1, '0', '1']`.
      * @return bool Success
      */
-    public static function boolean($check)
+    public static function boolean($check, array $booleanValues = [])
     {
-        $booleanList = [0, 1, '0', '1', true, false];
+        if (!$booleanValues) {
+            $booleanValues = [true, false, 0, 1, '0', '1'];
+        }
+
+        return in_array($check, $booleanValues, true);
+    }
+
+    /**
+     * Truthy validation, determines if value passed is truthy.
```

Comment:
"Determines if the value passed is truthy."
or
""Determines if the value passed is truhy through a truthy validation."
(Although, I personally see no added information through stating "thruthy validation")